src/modules/theme/components/announcements/announcement-card.component.html

```diff
@@ -1,9 +1,24 @@
 <ng-container *ngIf="announcementCardData">
   <div class="nhsuk-card">
     <div class="nhsuk-card__content">
-      <h2 class="nhsuk-card__heading nhsuk-heading-xs text-pre-line text-word-break">
-        {{ announcementCardData.title }}
-      </h2>
+      <div class="d-flex justify-content-space-between nhsuk-u-margin-bottom-1">
+        <div>
+          <h2 class="nhsuk-card__heading nhsuk-heading-xs text-pre-line text-word-break">{{ announcementCardData.title }}</h2>
+        </div>
+        <div *ngIf="isDashboardPage || isOverviewPage" class="width-15 nhsuk-u-font-size-19 text-align-right">
+          <a href="javascript:void(0)" (click)="clearAnnouncement()">Clear</a>
+        </div>
+      </div>
+      <p *ngIf="announcementCardData.innovations && !isOverviewPage" class="font-color-secondary">
+        <span
+          >{{ "Re:" }}
+          <ng-container *ngFor="let innovation of announcementCardData.innovations; let first = first; let last = last">
+            <span *ngIf="!first && !last">{{ ", " }}</span>
+            <span *ngIf="last">{{ " and " }}</span>
+            <span>{{ innovation }}</span>
+          </ng-container>
```

Comment:
I don't really like us doing this here. Would prefere to either:
- Do this on the TS, and create a variable with it.
- Create a pipe that will contain this logic and can be reused. I can see this be used in other places. Would contain the logic to display:
    - [A] -> A
    - [A,B] -> A and B
    - [A,B,C] -> A, B and C

Reply:
Yes sounds like a good idea, will do the TS version, and later try to create a pipe, I think we do have other cases of similar cases.